test/exportToNDJson.test.js

```diff
@@ -0,0 +1,41 @@
+const build = require('../src/server/app');
+const { exportToNDJson } = require('../src/util/exportToNDJson');
+const { cleanUpDb, createTestResourceWithConnect } = require('./populateTestData');
+const testPatient = require('./fixtures/testPatient.json');
+const app = build();
+const fs = require('fs');
+const mockRequestWithType = {
+  query: {
+    _type: 'Patient'
+  }
+};
+const mockRequestWithoutType = {
+  id: 'mockRequestWithoutType',
+  query: {}
+};
+
+const expectedFileName = './tmp/123456/patient.ndjson';
```

Comment:
```suggestion
const expectedFileName = './tmp/123456/Patient.ndjson';
```

pretty sure it will take the resource type directly so you might run into a capitalization issue here

Reply:
actually looks like this doesn't cause an issue lol. I guess fs not case sensitive